sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -88,10 +88,14 @@ class SubscriptionsDbConnector implements SubscriptionDataSource {
     }
     
     private void addSubscription(Connection connection, int subscriberId, String projectCode) {
+
+          if (projectAlreadySubscribed(subscriberId, projectCode)) {
```

Comment:
I would expect something like this: if not in DB already then add the subscription

```suggestion
          if (!projectAlreadySubscribed(subscriberId, projectCode)) {
```

Reply:
Done!